commands/base_commands/guest.py

```diff
@@ -90,6 +90,41 @@
     "Valardin",
     "Velenosa",
 )
+_valid_religion_ = (
+    "Pantheon",
+    "Shamanism",
+    "Pantheon/Shamanism",
+)
+_valid_hair_color_ = (
+    "Red",
+    "Brown",
+    "Black",
+    "Blonde",
+    "Gray",
+    "White",
+    "Bald",
+)
+_valid_skin_tone_ = (
+    "Pale",
+    "Fair",
+    "Tan",
+    "Black",
+    "Bronzed",
+    "Brown",
+    "Porcelain",
+    "Smoke",
+    "Sunkissed",
+    "Olive",
+    "Dusky",
+)
+_valid_eye_color_ = (
+    "Brown",
+    "Blue",
+    "Green",
+    "Hazel",
+    "Gray",
+)
```

Comment:
This is an anti-pattern that was used early on and is being continued here - ideally, all these values should live in the database so can be controlled by staff at will. We can hold off on that implementation until CG is reworked, however, because you'd probably want to do it via the web and it's much easier to have a form or serializer use options based on a query than it is to do the same thing here

Reply:
Yeah that makes sense, we'd make a CG character set on django and then have it pull from that, got it.